frappe/public/js/frappe/model/create_new.js

```diff
@@ -137,15 +138,17 @@ $.extend(frappe.model, {
 				return user_permissions[df.options][0];
 			}
 
-			// 2 - look in user defaults
-			var user_default = frappe.defaults.get_user_default(df.options);
+			if(!df.ignore_user_permissions) {
+				// 2 - look in user defaults
+				user_default = frappe.defaults.get_user_default(df.options);
 			
-			if (!user_default) {
-				user_default = frappe.defaults.get_user_default(df.fieldname);
-			}
+				if (!user_default) {
+					user_default = frappe.defaults.get_user_default(df.fieldname);
```

Comment:
Fieldname also takes from user permissions
Have checked that default value is fetching for the field